converters/video_to_video/video_to_video.py

```diff
@@ -41,5 +49,24 @@ def convert(self, output_format=None, **kwargs):
             ffmpeg_command.run(overwrite_output=True)
             return output_path
         except ffmpeg.Error as e:
-            print(f"Error executing ffmpeg command: {e}")
-            raise
+            raise VideoConvertError(f"Ffmpeg command execution failed: {e.stderr}" , 500)
+
+    def validate_params(self, **kwargs):
+        validators = [ FormatValidator(kwargs['output_format'], VIDEO_OPTIONS['format'], "Output format") ]
+    
+        if kwargs['fps']:
```

Comment:
We could validate a minimum fps of 1, although as a suggestion because when I try with 0 I get an error message that also works.
![image](https://github.com/user-attachments/assets/eb4a5fae-f6c8-49b2-b74e-15868d9ee210)